addon/authenticators/jwt.js

```diff
@@ -120,11 +122,11 @@ export default TokenAuthenticator.extend({
 
```

Comment:
Ahh, of course I see what you mean now. Yeah, you're right, if your token is already expired it won't be refreshed.

I think the change here would be if it's expired to attempt a refresh, if the server rejects the refresh token then unauth the session so they are sent to the login form.